pyproject.toml

```diff
@@ -59,6 +59,8 @@ pytest-xdist = ">=2.5.0"
 types-backports = ">=0.1.3"
 types-pyyaml = ">=6.0.4"
 types-psutil = "*"
+types-pygments = ">=2.17"
+types-colorama = ">=0.4"
```

Comment:
Would you mind sorting the dependencies? I just noticed that `types-pyyaml` was already in the "wrong" place, too. I'd say, just sort them all now.